marklogic-data-hub/src/main/resources/hub-internal-artifacts/step-definitions/mapping/marklogic/entity-services-mapping.step.json

```diff
@@ -9,6 +9,7 @@
   "threadCount" : 4,
   "options": {
     "acceptsBatch": true,
+    "attachSourceDocument": true,
```

Comment:
@rjrudin ,
I have added this here so that steps without this property or in case of upgrade to 5.5, attachment will still be present. Let me know  if that's ok.

Reply:
That sounds good to me